devenv/scripts/deploy_e2e_tester.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env sh
+set -e
+CLUSTER_RESOURCE_GROUP=$(cat devenv/state/cluster-info.json | jq '.ClusterResourceGroup' | tr -d '"')
+CLUSTER_NAME=$(cat devenv/state/cluster-info.json | jq '.ClusterName' | tr -d '"')
+
+echo "adding image tag to kustomize and generating configmap..."
+cp devenv/kustomize/* devenv/state/kustomize
+cd devenv/state/kustomize
+kustomize edit set image placeholderfortesterimage=`cat ../e2e-image-tag.txt`
+
+echo "deleting any existing e2e job..."
+az aks command invoke --resource-group $CLUSTER_RESOURCE_GROUP --name $CLUSTER_NAME --command "kubectl delete jobs app-routing-operator-e2e -n kube-system --ignore-not-found && kubectl apply -k ." --file .
```

Comment:
good to just separate the delete and creation parts into two separate invokes. It would be more obvious what was failing if one did fail that way

Reply:
I'm not getting the status of the command invoke command. The status that I'm getting is what's output from the JSON describing the pod

Reply:
The statuses do matter because of `set -e`.

Reply:
modifying accordingly after our discussion!